lib/langParser.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs')
+const iso639 = require('iso-639-1')
+
+fs.readdir(`${__dirname}/../static/i18n`, function(err, items) {
+  var avaliableLanguages = {
+    Languages: {},
+  }
```

Comment:
Does this need to have a nested object? Would be much cleaner to just use `availableLanguages = {}`

Reply:
It needs to be so that we can import all nested objects at once.